packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -401,7 +401,7 @@ Based on the type of token and whether it is active, the returned JSON contains
 | Error Id          | Details                                                                                                  |
 | :---------------- | :------------------------------------------------------------------------------------------------------- |
 | invalid_client    | The specified `client_id` wasn't found.                                                                  |
```

Comment:
General comment: Should the sentences in the Details column be in the present tense?
wasn't found > isn't found
was invalid > is invalid